src/containers/quiz/quiz-header/QuizHeader.tsx

```diff
@@ -33,22 +46,39 @@ const QuizHeader = ({
   isGraded,
   questionsAnswered,
   createdAt,
-  updatedAt
+  updatedAt,
+  isNotStarted,
+  quizItems,
+  settings,
+  handlePreview
 }: QuizHeaderProps) => {
   const { userRole } = useAppSelector((state) => state.appMain)
-
   const isStudent = userRole === UserRoleEnum.Student
 
   const isTutor = userRole === UserRoleEnum.Tutor
+  const { id, quizId } = useParams()
+
+  const getQuizzes = React.useCallback(() => {
+    if (quizId && id) {
+      return ResourceService.getFinishedQuizzesByQuizId(id, quizId)
+    }
+  }, [id, quizId])
 
+  const { data: finishedQuizzes, isLoading } = useQuery({
+    queryKey: ['quizzes', id, quizId],
```

Comment:
What does this `id` mean? Is that id of cooperation, or quiz? Add better naming inside usage of `useParams`, please
```suggestion
    queryKey: ['quizzes', id, quizId],
```

Reply:
thanks, it is cooperation id